exampleradialbuild.yaml

```diff
@@ -1,6 +1,27 @@
-# Tokamak radial build, radially outward from plasma
-# Format: <material_region>: <thickness [cm]>
+# A tokamak radial build, ordered radially outward from the plasma. If differentiation
+# between the inboard and outboard is desired, separate lists of the desired material
+# layers must be made for each, headed by "Inboard" and "Outboard". Otherwise, a single
+# list represents a tokamak that is radially symmetric.
+
+# The example given below differentiates between the inboard and outboard.
+
+# Format
+# ------
+# <region>:
+#   <material_layer>: <thickness [cm]>
 ---
-fw: 5
```

Comment:
Don't you want to keep some example of this structure as well?  It is still supported in your code, so you probably want something for testing, as well.

Reply:
good point! what do you think about inserting an example symmetric build just as comments above "Inboard"? could also just include a separate yaml file. does your expertise say one option is better than the other?

Reply:
Recommend a separate yml file